### PR TITLE
[PUBLIC-1668] Remove ConditionSet

### PR DESCRIPTION
**Pull request by ghost**
Remove ConditionSet from "Create RNA-seq Sample Set" UI.
https://kbase-jira.atlassian.net/browse/PUBLIC-1668

### CODE REVIEW

File: lib/SetAPI/sampleset/SampleSetInterface.py [hunk 1]
```diff
@@ -53,11 +53,6 @@ def create_sample_set(self, ctx, params):
 
         pprint(params)
 
-        conditionset_ref = params.get('conditionset_ref')
```
Comment (Tianhao-Gu) [inline lib/SetAPI/sampleset/SampleSetInterface.py:56]: removing condition_set from UI is fine. But can we keep this just in case?

**Review by Tianhao-Gu — APPROVED** [review]
looks good. Just one comment whether we should only remove the UI component or completely remove the condition set from code.

File: lib/SetAPI/sampleset/SampleSetInterface.py (hunk 1 above)
Comment (ghost) [reply lib/SetAPI/sampleset/SampleSetInterface.py:56]: Maybe comment it out? I didn't want to leave too much unused stuff lying around that people might waste time trying to understand and get confused by.

Comment (ghost) [reply lib/SetAPI/sampleset/SampleSetInterface.py:56]: ConditionSet doesn't exist I think, and it sounds like it isn't really a priority

Comment (Tianhao-Gu) [reply lib/SetAPI/sampleset/SampleSetInterface.py:56]: 👍  It only exists in CI.

**Comment by Tianhao-Gu** [comment]
💯 LGTM.

**Pull request merged**